environment.yml

```diff
@@ -0,0 +1,13 @@
+name: cta
+channels:
+  - default
+  - cta-observatory
+  - conda-forge
+dependencies:
+  - python>3.5
+  - ctapipe
+  - gammapy
+  - pip:
+      - pytest_runner
+      - pytest-ordering
+      - https://github.com/cta-sst-1m/protozfitsreader/archive/v1.4.2.tar.gz
```

Comment:
Here, as discussed in #2, I think we should also depend on `ctapipe-extra`, so that the geometry files of the different cameras are known.

Reply:
The dependence from ctapipe-extra is already set by the ctapipe installation

Reply:
This is related to my last reply above: if `ctapipe` is installed via `pip` in this conda environment file, then a dependency on `ctapipe-extra` should be added here, presumably installing via `pip` too. If we stick with the conda installation of `ctapipe`, then I agree with you. But in the later case, `ctapipe` would be installed twice: once via conda, and a second time through the pip installation. Isn't there a risk of conflicting versions in such a case ?